frontend/public/components/catalog/catalog-item-details.jsx

```diff
@@ -45,7 +45,9 @@ export class CatalogTileDetails extends React.Component {
 
     const creationTimestamp = _.get(obj, 'metadata.creationTimestamp');
 
-    const supportUrlLink = <ExternalLink href={supportUrl} text="Get support" />;
+    const supportUrlLink = (
+      <ExternalLink href={supportUrl} text="Get Support" />
```

Comment:
I'd revert this change. There shouldn't need to be any changes in this file now that we're passing the right `supportUrl`

Reply:
I'll remove the linter parenthesis, but I'd argue "Get Support" should be the same case throughout (it would be "Get Support" everywhere but this file if I revert everything).

Reply:
Everywhere else it's `Get support`, so I'd recommend changing the new link to match instead of changing this one. That's more inline with the Patternfly recommendations and other similar links in console like `View events`, `View alerts`, etc.

https://github.com/openshift/console/search?q=%22get+support%22&unscoped_q=%22get+support%22